### PR TITLE
added support for pythia 70M model fine-tuning saves model

### PR DESCRIPTION
**Pull request by ajn2004**
I believe this adds pythia functionality for our current branch. There were changes to the quickstart files for testing.
Completion and finetuning work on my machine with this model, using chat_gpt as a teacher.

### CODE REVIEW

File: src/llm_vm/onsite_llm.py [hunk 1]
```diff
@@ -66,7 +67,99 @@ def generate(self): # this is where the meat and potatoes should live?
...
+        tokenizer (AutoTokenizer): Tokenizer from Transformer's library
+        model (LLM): The large language model
+
+    Methods:
+        model_loader: Loads the LLM into memory
+        tokenizer_loader: Loads the tokenizer into memory
+        generate: Generates a response from a given prompt with the loaded LLM and tokenizer
+    """
+    
+    def __init__(self,model_uri_override="EleutherAI/pythia-70m-deduped"): # tokenizer_kw_args=None,model_kw_args=None
```
Comment (cartazio) [inline src/llm_vm/onsite_llm.py:86]: We should remember to rename this and maybe merge in the partial doc branch?

Comment (ajn2004) [reply src/llm_vm/onsite_llm.py:86]: I changed the name to model_uri, we'll need to recode the workflow for these classes to incorporate parameter setting for different sized models.

File: quickstart_finetune.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 from llm_vm.client import Client
 import os
 # Instantiate the client specifying which LLM you want to use
-client = Client(big_model='gpt', small_model='opt')
+client = Client(big_model='chat_gpt', small_model='pythia')
```
Comment (cartazio) [inline quickstart_finetune.py:5]: How fast is fine tuning ?

**Comment by cartazio** [comment]
Seems fine to me. Once training fine tuning is validated

**Pull request merged**